notes/2019-12-05.md

```diff
@@ -0,0 +1,227 @@
+*   Lee: I think there are two paths here and they have different implications/problems. 1) revisit the RFC that you closed earlier about supporting empty object types, changing the grammar to allow empty selections and empty objects types. 2) Allow there to be no Query type. Might be strange but it might make sense in some use cases like these micro GraphQL endpoints you pointed out. 
+*   Need to look through the spec, and fiddle with some implementation to figure out if we did this, what would it mean to implement an introspection query (a query of meta fields) against a schema with no Query type. 
+*   Victor: That makes sense. I think that is a reasonable analysis of the situation. To be fair (the one driving this), I just brought up this problem because I thought it was interesting. I can’t hide my bias towards the first suggestion. 
+*   Lee: Ultimately, you’re the champion so you can decide which solution to pursue. 
+*   [ACTION: Michael] Document this feature in a PR to keep the discussion going
+
+## Crowd-funding GraphiQL
+*5m, Benjie*
+
+### Context
+
+*   Issue: [graphql/graphiql/issues/1067](https://github.com/graphql/graphiql/issues/1067)
+*   Funding: [funding.communitybridge.org/projects/graphql](https://funding.communitybridge.org/projects/graphql)
+
+### Notes
+
+*   Benjie: Working hard on GraphiQL… Struggling to keep up as a group. Trying to bring along other maintainers but what we want to do is to enable greater functionality. Playgrounds, integration with VS Code and Vim, we want to make it extensible… But we are struggling with human resources to do that. 
+*   Benjie: I’ve been speaking to Chris with the Linux Foundation about crowd funding
+*   Benjie: Just wanted to let people be aware of that
+*   Lee: Are you thinking of from the current set of maintainers, just setting up official contracts so that they spend a certain number of hours working on GraphiQL? Or hiring new people?
+*   Benjie: I was thinking more like how Open Collective operates, there’s a group the can decide to fund someone to complete a certain task. 
+*   Benjie: It does not need to be just maintainers, it could be open to people in the community
+*   Lee: So Open Collective works more like a bounty program, right?
+*   Benjie: Yeah, I think so.
+*   Lee: One thing I’d like to talk to you (Benjie) about moving funds from the GraphQL foundation.
+*   Benjie: Yeah, that sounds great! I think also just encouraging people to help out on GraphiQL is great. 
+*   Lee: What are some easier ways to contribute to those pools? Maybe just setting up Open Collective or setting up GitHub sponsors. 
+*   Ivan: Set up under Linux Foundation or Kubernetes Foundation or Open Collective and reuse the experience. We need to ask Chris. 
+*   Lee: I know the JS Foundation has something set up too. I will ask them as well. 
+*   [ACTION: Lee] Ask the JS Foundation how they set up funding.
```

Comment:
we already have funding setup that we have been using with Ivan, LF cover all the fees and anyone can contribute
https://funding.communitybridge.org/projects/graphql